po/uk.po

```diff
@@ -1192,54 +1192,54 @@ msgstr[2] "доповнено %d локальними об’єктами"
 #: builtin/index-pack.c:1300
 #, c-format
 msgid "Unexpected tail checksum for %s (disk corruption?)"
-msgstr ""
+msgstr "Неочікувана контрольна сума наприкінці %s (пошкодження диска?)"
 
 #: builtin/index-pack.c:1304
 #, c-format
 msgid "pack has %d unresolved delta"
 msgid_plural "pack has %d unresolved deltas"
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "пакунок має %d невирішену дельту"
```

Comment:
```suggestion
msgstr[0] "пакунок має %d нерозпізнану дельту"
```

Not sure if this option is better 🤔. It is a bit inconsistent at this moment.

Reply:
Nope 🙂

I see 2 different resolution types here:
  - resolve an entity to another entity (name, path)
  - resolve a conflict, code inconsistency, delta

May be a good idea to revisit these cases later.